templates/common/_base/units/ovs-notification.yaml

```diff
@@ -0,0 +1,21 @@
+name: ovs-notification.service
+enabled: {{if eq .NetworkType "OVNKubernetes"}}true{{else if eq .NetworkType "OpenShiftSDN"}}true{{else}}false{{end}}
+contents: |
+  [Unit]
+  Description=Creates a file to let the OVS pod know OVS is running on systemd.
+  # Don't write the notification before OVS starts
+  Requires=openvswitch.service
+  Wants=NetworkManager-wait-online.service
+  After=NetworkManager-wait-online.service openvswitch.service
+  Before=network-online.target kubelet.service crio.service node-valid-hostname.service
+
+  [Service]
+  # Need oneshot to delay kubelet
+  Type=oneshot
+  # Create the file in a tmpfs so that the file gets deleted on restart
+  ExecStart=/usr/bin/touch /var/run/openvswitch/ovs-notification
+  StandardOutput=journal+console
+  StandardError=journal+console
+
+  [Install]
+  WantedBy=network-online.target
```

Comment:
why?

Reply:
This is part of the network so I'd say we can't consider the network completely setup until this unit is complete. I would also make sense in the network.target but I think network-online.target is the most logical place.

Anyway I don't feel strongly enough about this to block the PR while we discuss where it should go, so if you want it somewhere else I'll move it.

Reply:
This isn't right to say wanted by network online target, but then in runtime this is running before network online target